src/index.js

```diff
@@ -17,6 +18,7 @@ function Router() {
                 <Route exact path="/" component={Index} />
                 <Route exact path="/community" component={Community} />
                 <Route exact path="/resources" component={Resources} />
+                <Route path="/profile/:alias" component={Profile} />
```

Comment:
What happens if an alias isn't provided for whatever reason?

Reply:
It will 404, currently just displays a blank page as no 404 page has been set up, however.